.github/workflows/release.yml

```diff
@@ -29,12 +29,6 @@ jobs:
             TAG=${GITHUB_REF#refs/tags/}
             echo ::set-output name=tag_name::${TAG}
 
-      - name: Login to Docker Hub
```

Comment:
We shouldn't revert changes in this file for go 1.19.

Reply:
reverted